grammatical and info fixes on roundstart tips + adds a single space to the valhalla mobspawn message

coldironwarrior opened the pull request:
## About The Pull Request

does some edits on the roundstart tips for syntax + grammar; removes a few that are no longer true, as far as I can tell.

adds a space to the long message you get when entering valhalla about how rules still apply, etc.

## Why It's Good For The Game

keeps them kind of useful and fun for people; there may be some worthwhile stuff in here that people can learn things from and i'd hate to have just downright outdated and untrue things mixed in still

that missing space really bothered me!!!

## Changelog


:cl:
spellcheck: updates the roundstart tips
/:cl:

Lumipharon commented:
some of these tip changes are just wrong. Like more than runner can leap over cades etc.

coldironwarrior commented:
> some of these tip changes are just wrong. Like more than runner can leap over cades etc.

we can re-work the wording of specific ones that strike you as factually wrong if you'd like to list them out.

amended the one you pointed out to include hunters as well, as runners and hunters are the two castes with the ability to "pounce", as the wording of the tips evokes to me.

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -1,59 +1,56 @@
 As a hunter, the movement of your Mirage is controlled by your intent.
 Defenders have a very small window to sweep grenades back at marines with their tail.
 Aiming for the head is a good long-term investment, causing brain damage on marines.
-Aiming for the groin can easily damage someones liver.
```
this is still true

coldironwarrior replied:
liver is a part of the chest, do you want it just changed to "aiming for the chest"?

/code/modules/organs/organ_internal.dm
```/datum/internal_organ/liver
	name = "liver"
	parent_limb = "chest"```

Lumipharon replied:
yep. Probably expand it to the other organs there.

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -1,59 +1,56 @@
-As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barricades and defenses, melt or constantly spit acid on them if you can.
-As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
-As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
+As a ravager, the King or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it.
```
king is actually fireproof, unless they get sundered.

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -1,59 +1,56 @@
-As a larva, stay on weeds while growing, you will grow faster as a result.
-As a runner or larva, you can pass over window frames without climbing on to them beforehand.
+As a larva, stay on weeds while growing. You will grow faster as a result.
+As a runner or larva, you can pass over window frames without climbing onto them beforehand.
```
also warlock

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -1,59 +1,56 @@
 As a xeno, do not block other xenos' paths when they are retreating. Circumvent this by being on Help Intent or by simply moving away.
-As a xeno, white weed helps you heal faster, green weeds slows down marines, and weed with no colors are normal weeds. Rest on the white weeds to heal faster!
-As a larva, ask your hive leader what you should evolve into, if nobody responds you can check hive status and think what xenomorphs need the most.
+As a xeno, white weed helps you heal faster, blue weeds slows down marines, and weeds with no colors are normal weeds. Rest on the white weeds to heal faster!
```
Healing weeds are green

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -1,59 +1,56 @@
 As a xeno, you've got pheromones that can help the hive.
-As a xeno, you can pounce past window frames and barricades without barbed wires.
-As a xeno, pouncing into a barricade with barbed wire stuns you.
+As a runner or hunter, you can pounce past window frames and barricades without barbed wires.
```
Many xenos can leap over barricades. Defender, warrior, prae, ravager off the top of my head.

Lumipharon commented on strings/tips/meta.txt:
```diff
@@ -1,25 +1,24 @@
 To sing, add "%" before any text. Singing on radios will not work, but singing on intercomms will.
 You can respawn during End-Of-Round Deathmatch in the OOC tab.
-Sundering is a concept that reduces the effectiveness of xeno armor by percentage. Xenomorph drones can heal and decrease sunder. Resting heals sunder, but slowly.
+Sundering reduces the effectiveness of xeno armor by percentage. Xenomorph drones can heal and decrease sunder. Resting heals sunder, but slowly.
```
there's a few mentions like this one that list some but not all xenos that can do X. Either it should list them all or just be more general. "some castes can do..." "building castes should..." etc

Lumipharon commented on strings/tips/meta.txt:
```diff
@@ -1,25 +1,24 @@
+When in doubt: Adminhelp for problems with people violating the rules, Mentorhelp for questions that you want to know the answer to.
 Check the SS13 Webmap to learn the layout of the ship or the map currently in play.
-After you ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a human) or 2 minutes (as a xenomorph) if there is a larva.
+After you have ghosted out of the game (either from dying, cryo or manual ghost), you can respawn back into the game after waiting for 30 minutes (as a marine) or 2 minutes (as a xenomorph) if there is a larva available.
```
I don't think the 2 minute thing exists for xenos.

Lumipharon commented on strings/tips/marine.txt:
```diff
@@ -109,30 +107,27 @@ Chemrette cigarettes can be very useful.
 Switching to your sidearm is faster than reloading.
 You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload.
+With your sidearm in hand, click a pistol pouch to instantly reload.
```
it specifically needs to be right mouse button. Otherwise you're just storing it.

Lumipharon commented on strings/tips/marine.txt:
```diff
@@ -85,21 +84,20 @@ As an engineer, placing turrets on open spaces and without defenses is a poor ch
 Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
 Don't forget that xenomorphs spawn infinitely during the Crash gamemode. Don't waste too much time hunting xenomorphs down!
-The AI and synthetics can talk together on their own channel using the :n key.
-You can memorialize fallen marines by using their dogtags on the ship's memorial! Memorialized solders will be displayed at the end of the round.
```
?

coldironwarrior replied:
will fix the :n --> .n back to the :n.

unless you meant the memorial thing; that hasn't worked in months.

Lumipharon commented on strings/tips/marine.txt:
```diff
@@ -1,80 +1,79 @@
-As a human, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself on them.
+Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.
+Cargo points are valuable: some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
+As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself onto them.
```
should probs mention jumping.

coldironwarrior replied:
i went ahead and made it a new tip, just because i think combining the two would be too long, and i wanted to flesh out more than just "you can also jump". i can also just delete the "specialclicking" tip since i think jumping is the superior way to move across objects

```You can jump over waist-high obstacles like platforms, tables, and windowframes, for a small stamina cost, or, if a xeno, for a two-second cooldown. Set your key to jump under the Game Preferences menu!```

let me know if you want the wording edited at all. i also forgot, but it has to be binded, there's no default key, correct?

Lumipharon replied:
but default it should be space or something.

Lumipharon commented on strings/tips/xeno.txt:
```diff
@@ -12,26 +12,26 @@ The queen, shrike, and hivelord have access to Queen's Blessing, a menu for the
-As a runner or hunter, you can pounce past window frames and barricades without barbed wires.
-As a runner or hunter, pouncing into a barricade with barbed wire stuns you.
+As a xeno, you can pounce past window frames and barricades without barbed wires.
+As a xeno, pouncing into a barricade with barbed wire stuns you.
```
pretty sure you just bounce off.

coldironwarrior commented:
probably worth mention that addressed reviews aside, i also added in a tip for xeno.txt that some xenos have the ability to recycle their fallen hive members to deny marines requisition points.

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -1,80 +1,80 @@
-Cargo points are valuable: Some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
-As a human, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself on them.
+Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.
+Cargo points are valuable: some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
```
I don't think we even have these in req anymore

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -1,80 +1,80 @@
+You can remove armor pieces (leg pieces, arm pieces and chest pieces) and armor modules (Valkyrie, Baldur, etc.) from the XM-02 Combat Exoskeleton by Alt+clicking the exoskeleton.
 You can customize armor pieces and the exoskeleton helmet using any kind of facepaint for a personal touch. Paint the armor pieces before you put them in the exoskeleton!
-If a M40 FLDP grenade collides to any mob when thrown, the mob will be lit in a light fire. Be careful in throwing flares.
+If an M40 FLDP grenade collides into any mob when thrown, the mob will be lit in a light fire. Be careful throwing flares.
```
```suggestion
If an M40 FLDP grenade collides with any mob when thrown, the mob will be lit in a light fire. Be careful throwing flares.
```

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -1,80 +1,80 @@
 SWAT masks ordered from Requisitions can block only one larval facehugger attack. If they successfully blocked an attempted attack, replace them immediately.
-Drag crates of platinum and phoron to the Automated Storage and Retrieval System (ASRS) pad inside cargo. Once the pad lowers with the applicable crates, you will gain requisition points. This applies to dead xenomorphs as well!
-In snow maps (such as Ice Colony or Icey Caves), wear a coif or scarf! You will freeze to death if you go outside of your dropship if you are not wearing them!
+Drag dead xenomorphs onto the Automated Storage and Retrieval System (ASRS) pad inside cargo. Once the pad lowers with the applicable bodies, you will gain requisition points.
```
```suggestion
Drag dead xenomorphs onto the Automated Storage and Retrieval System (ASRS) pad inside requisitions. Once the pad lowers with the applicable bodies, you will gain requisition points.
```

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -1,80 +1,80 @@
 As a squad marine, you can help out your squad by bringing extra ammunition and food to share.
 As a squad marine, you can take two two-handed guns, one in the suit slot and one on your back.
-As a squad marine, you can get two attachments of your choice from GHMME Automated Closet. Though you can self-serve to an attachments vender usually near preperations, which has no limit on how many attachments you can vend.
+As a squad marine, you can get attachments from the weapons vendors, usually near preparations.
```
```suggestion
As a squad marine, you can get attachments from the weapons vendors, usually in preparations.
```

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -85,21 +85,21 @@ As an engineer, placing turrets on open spaces and without defenses is a poor ch
 Don't forget that xenomorphs spawn infinitely during the Crash gamemode. Don't waste too much time hunting xenomorphs down!
-The AI and synthetics can talk together on their own channel using the :n key.
-You can memorialize fallen marines by using their dogtags on the ship's memorial! Memorialized solders will be displayed at the end of the round.
+The AI and synthetics can talk together on their own channel using the :n prefix.
```
```suggestion
The AI and synthetics can talk together on their own channel using the :n or .n prefix.
```

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -109,30 +109,27 @@ Chemrette cigarettes can be very useful.
 Depending on the tactical situations, smoke grenades can do more harm than good.
 A miner without a turret is a sitting duck.
-Claymores often 1-shot T1s.
+Claymores often one-shot T1s.
```
This one isn't even true.

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -144,16 +141,16 @@ A single wraith can be fatal to the operation.
 Try not to bump into corpsmen. Your life might depend on it.
 With a cigarette pack in hand, target the mouth and click yourself to instantly take one.
-Spamming flares can be incredibly helpful, especially when King use Nightout! Many marines cannot see in darkness, so if you lighten up the firing lane with flares, marines can shoot better.
+Spamming flares can be incredibly helpful, especially when Kings use Nightfall! Many marines cannot see in darkness, so if you lighten up the firing lane with flares, marines can shoot better.
```
```suggestion
Spamming flares can be incredibly helpful, especially when the King uses Nightfall! Many marines cannot see in darkness, so if you lighten up the firing lane with flares, marines can shoot better.
```

ivanmixo commented on strings/tips/marine.txt:
```diff
@@ -144,16 +141,16 @@ A single wraith can be fatal to the operation.
 The cryopods in medbay heal most types of damage but not organ damage!
 When in doubt, throw them in the autodoc.
-Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors or replace the power cell in medbay APC.
+Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors or replace the power cell in medbay's APC.
```
```suggestion
Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors or replace the power cell in the medbay APC.
```

ivanmixo commented on strings/tips/meta.txt:
```diff
@@ -1,25 +1,24 @@
 When you want to claim a mob, claim it now! Others will claim it as soon as they are available.
 You can customize your ghost using the Ghost Customization command under the Preferences tab.
-As either a marine or a xeno, you will die quickly and horribly if you find yourself in a pressureless enviroment (a.k.a spess or water).
+As either a marine or a xeno, you will die quickly and horribly if you find yourself in a pressureless enviroment (a.k.a spess or underwater).
```
Where do we even have underwater?

coldironwarrior replied:
real. why was water included in the tip in the first place. we may never know.

ivanmixo commented on strings/tips/xeno.txt:
```diff
@@ -1,66 +1,65 @@
+As a xeno, you can freely melt barricades while in boiler or defiler gas as long as you can click on the right spot. The gas is also a really good hiding spot to launch ambushes from without marines being aware of you.
 As a xeno, always thank your drones and hivelords for supporting the hive!
-As a xeno, you can drag any items, but you must wait before you drag the desirable item!
+As a xeno, you can drag any items, but you must wait through an actionbar before you can do so!
```
```suggestion
As a xeno, you can drag any item, but you must wait through an actionbar before you can do so!
```

ivanmixo commented on strings/tips/xeno.txt:
```diff
@@ -1,66 +1,65 @@
 If you have acid spit, you can spit at a welding fuel tank to blow it up. If the welding fuel is near hostiles, you might give them a pleasant surprise!
 As a xeno, check hive status often.
-As a xeno, you've got pheromones that can help the hive.
+Some xenos have pheromones that can help the hive. Frenzy increases speed, Warding increases armor and armor healing, and Recovery increases plasma and health healing.
```
Warding speeding up armor healing is just a myth. Recov also doesn't help with plasma recovery.

ivanmixo commented on strings/tips/xeno.txt:
```diff
@@ -1,66 +1,65 @@
-As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
+As a ravager or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it. As the King, however, you're fireproof until your armor is damaged by sunder.
+As a spitter, boiler, praetorian, or the Queen, take advantage of unguarded barricades and defenses; melt or constantly spit acid on them if you can.
+As a boiler, the King or the Queen, you have the most powerful corrosive acid for melting things.
```
Primordial hivelord also gets very strong acid

ivanmixo commented on strings/tips/xeno.txt:
```diff
@@ -1,66 +1,65 @@
 As a boiler, you'll begin to glow with the more globules you have stored.
 If you see a red dot, RUN!
 Allowing marines to push caves only to block off their exit is hillariously effective.
-As a xeno, a Tier 3 unit takes up both a Tier 3 and Tier 2 slot.
```
This is true, why'd you delete?

coldironwarrior replied:
i checked the code but didn't see anything for it, will add back
